packages/node/src/integrations/tracing/connect.ts

```diff
@@ -48,12 +50,19 @@ export const setupConnectErrorHandler = (app: ConnectApp): void => {
     });
   }
 
-  if (!isWrapped(app.use) && isEnabled()) {
+  if (!isWrapped(app.use) && isEnabled() && hasTracingEnabled()) {
```

Comment:
Could we maybe extract this into a utility and re-use this? Could be something like this:

```ts
function ensureIsWrapped(maybeWrappedModule: unknown, name: string): void {
  if (!isWrapped(maybeWrappedModule) && isEnabled() && hasTracingEnabled()) {
    // ...
  }
}
```